SETUP.md

```diff
@@ -131,30 +131,50 @@ SPARK_WORKER_OPTS="-Dspark.worker.cleanup.enabled=true, -Dspark.worker.cleanup.a
 
 ## Setup guide for Azure Databricks
 
-### Requirements of Azure Databricks
+The setup for Azure Databricks can be completed by running the [create_and_configure_cluster.ipynb notebook](./notebooks/05_operationalize/create_and_configure_cluster.ipynb)
+on any jupyter instance that has the `requests` python module installed. It requires a few additional parameters to be configured; 
+please see the full details in that notebook.
+
+The following are the steps that notebook takes to prepare Azure Databricks.
+
+### Requirements for Azure Databricks
+
 * Runtime version 4.1 (Apache Spark 2.3.0, Scala 2.11)
 * Python 3
 
+**Please note:** This configuration is no longer available via the *Clusters* tab of the databricks interface, and you 
+**must** either use the API to create this configuration or clone a pre-existing cluster of this configuration. Please see the [setup notebook](./notebooks/04_operationalize/create_and_configure_cluster.ipynb) for details.
+
 ### Repository upload
+
 We need to zip and upload the repository to be used in Databricks, the steps are the following:
-* Clone Microsoft Recommenders repo in your local computer.
-* Zip the contents inside the Recommenders folder (Azure Databricks requires compressed folders to have the .egg suffix, so we don't use the standard .zip):
-```
+
+* Clone [Microsoft Recommenders](https://github.com/Microsoft/Recommenders) repository to your local computer.
+* Zip the contents inside the Recommenders folder (Azure Databricks requires compressed folders to have the `.egg` suffix, so we don't use the standard `.zip` suffix):
```

Comment:
I think .zip works fine too (it doesn't need to be .egg), but that's a minor thing

Reply:
I couldn't upload the file as a .zip.